sympy/physics/mechanics/wrapping_geometry.py

```diff
@@ -83,11 +66,7 @@ def geodesic_length(self, point_1: Point, point_2: Point) -> ExprType:
         pass
 
     @abstractmethod
-    def _geodesic_end_vectors(
-        self,
-        point_1: Point,
-        point_2: Point,
-    ) -> tuple[Vector, Vector]:
+    def _geodesic_end_vectors(self, point_1, point_2,):
```

Comment:
Does this need to be public?

Reply:
I've made public in eab6a2408bfb9a20571aaf8f68dc2275453130a1.